jobs/build/build-sync/Jenkinsfile

```diff
@@ -196,6 +196,9 @@ node('covscan') {
         if (params.SKIP_MULTI_ARCH_PAYLOAD) {
             cmd << "--skip-multiarch-payload"
         }
+        if (params.TRIGGERED_FROM_GEN_ASSEMBLY) {
+            cmd << "--triggered-from-gen-assembly"
```

Comment:
Since now we don't need this, we can remove the flag and all its occurances in pipeline code

Reply:
done, and repushed